app/controllers/application_controller.rb

```diff
@@ -36,6 +36,12 @@ def welcome
     not_found
   end
 
+  def empty_state
```

Comment:
Loading the empty state setup from the server side seems unnecessarily complicated to me. See my comment in the empty state component please.